src/crawlee/_utils/models.py

```diff
@@ -9,7 +9,7 @@
 """Utility types for Pydantic models."""
 
 
-def _timedelta_to_ms(td: timedelta | None) -> Any:
+def _timedelta_to_ms(td: timedelta | None) -> float | None:
```

Comment:
We could be more strict and require float, but I have a feeling, that None could be used for something connected with defaults somewhere in code???

Reply:
I guess we can consider using only floats.

However, in `StatisticsState` and `StatisticsPersistedState`, we have fields like this:

```python
request_min_duration: Annotated[timedelta_ms | None, Field(alias='requestMinDurationMillis')] = None
```

Make sure it won't break them.

Reply:
I tried adding some tests (but not sure whether I should keep them) and I think it works as expected. I am not really so sure why the previous implementation allowed **any** in so many places in this file, so I am little afraid there is some non obvious edge case I could not think of...

Reply:
Well that can also break pydantic. See new commit that allows | None, but has to use Union due to known issues that Pydantic can have with from __future__ import annotations and | symbol not being available for older Python versions in type hints...

Reply:
Are you sure that the pipe syntax will work with Pydantic 3.9, for example?